### PR TITLE
service tests, h2 tests, mvc

### PR DESCRIPTION
**Pull request by u2107680007911543774e7r**

### CODE REVIEW

File: src/main/java/service/CustomerServiceImpl.java [hunk 1]
```diff
@@ -22,45 +24,43 @@
 	@Autowired
 	private CustomerRepository customerRepository;
 
-	@Override
```
Comment (SerhiiPasko) [inline src/main/java/service/CustomerServiceImpl.java]: This is bad. @Override must present

File: src/test/java/repository/CustomerRepositoryIntegrationTest.java [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+import config.DataConfig;
+import entity.Customer;
+import entity.Salesreps;
+
+@DirtiesContext
+@RunWith(SpringJUnit4ClassRunner.class)
+@ContextConfiguration(classes = DataConfig.class)
+@Transactional
+@Rollback
+public class CustomerRepositoryIntegrationTest {
```
Comment (SerhiiPasko) [inline src/test/java/repository/CustomerRepositoryIntegrationTest.java]: I wrote this type of test only as example

File: src/test/resources/scripts/schema.sql [hunk 3]
```diff
@@ -0,0 +1,16 @@
+CREATE SCHEMA IF NOT EXISTS MA_STUDENT;
+
+CREATE SEQUENCE IF NOT EXISTS CUSTOMER_SEQ
```
Comment (SerhiiPasko) [inline src/test/resources/scripts/schema.sql]: Haven't sence. You didn't need that, becouse you havent Id generator in Customer entity

**Review by SerhiiPasko — CHANGES_REQUESTED** [review]
Read comments

**Comment by Artsapollo** [comment]
Now looks good

**Review by Artsapollo — APPROVED** [review]
Now looks good

**Review by SerhiiPasko — APPROVED** [review]
Good

**Pull request merged**